string name
int32 ID
int32 totalfrontiers
float32 spanning_trees
float32 infogain
std_msgs/Header header
geometry_msgs/Pose2D frontier_loc
geometry_msgs/Pose2D robotxy_pos
geometry_msgs/Pose2D chosenfrontierxy_pos
#geometry_msgs/Pose[] waypoints


